src/cbmc/all_properties.cpp

```diff
@@ -219,7 +219,7 @@ void bmc_all_propertiest::report(const cover_goalst &cover_goals)
         if(g.second.status==goalt::statust::FAILURE)
         {
           jsont &json_trace=result["trace"];
-          convert(bmc.ns, g.second.goto_trace, json_trace);
+          convert(bmc.ns, g.second.goto_trace, json_trace, bmc.trace_options());
```

Comment:
My question still stands: why is this only being done for that particular invocation of `convert`, and not for the one in `bmc.cpp`?